src/core/platforms/android/androidplatformutilities.cpp

```diff
@@ -49,37 +49,50 @@ class FileCopyThread : public QThread
     Q_OBJECT
 
   public:
-    FileCopyThread( const QString &source, const QString &destination, Feedback *feedback )
+    FileCopyThread( const QString &source, const QString &appDataLocation, const QByteArray &gitRev, Feedback *feedback )
```

Comment:
Can we keep this generic (to allow copying any dir anywhere?

Reply:
@m-kuhn , how would you handle gitrev writing?

Reply:
I.e. adding gitRev (or "versionTag") is ok, making it specific for `appDataLocation` is the part that will make it trickier to separate demo_projects (need to be copied to a writable location) and other data (that only needs to be copied on android where we have a location that's not accessible for gdal, proj etc).